pkg/api.go

```
@@ -105,6 +109,34 @@ func (a API) GetAccount(foreignID string) (AccountPublic, error) {
 	return acc.GetPublicInfo(), nil
 }
 
+// Update any of the 'settings' fields on an Account
+func (a API) UpdateAccountSettings(foreignID string, update map[string]interface{}) (AccountPublic, error) {
+	// TODO: potato programming, needs cleanup
+	acc, err := a.Store.GetAccount(foreignID)
+	if err != nil {
+		return AccountPublic{}, err
+	}
+	for k, v := range update {
+		switch k {
+		case "PayoutAddress":
+			acc.PayoutAddress = v.(string)
+		case "PayoutThreshold":
+			acc.PayoutThreshold = v.(string)
+		case "PayoutFrequency":
+			acc.PayoutFrequency = v.(string)
+		default:
+			a.bus.Send(SYS_ERR, fmt.Sprintf("Invalid account setting: %s", k))
```

This probably also should return an error so the caller knows it failed.